src/components/language-selector.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { navigate } from "gatsby";
+import { Grid, Link, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import globe from "../images/logos/globe.png";
+
+function LanguageSelector() {
```

Comment:
I created language selector in another file but it will serve the same purpose as language switcher. I didn't want to interfere with our ability to select a language which working on this